install/configure.sh

```diff
@@ -6,6 +6,31 @@
 
 ROOT="$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )"
 
+echo ROOT=${ROOT}
+
+if [ ! -d "${ROOT}/.clean_" ]
+then
+    echo "Initializing CWS directory... "
+    mkdir ${ROOT}/.clean_
+    # Save clean CWS before configure.sh is run and directory is modified
+    cp -r ${ROOT}/* ${ROOT}/.clean_ && rm -rf ${ROOT}/.clean_/bpmn
+else
+    echo "Re-running CWS Config, backing up current configuration. Refer to .backups_ folder for all backups."
+
+    # Make backup_ folder for current CWS property version
+    BACKUP_DIR=${ROOT}/.backups_/.backup_$(date '+%F_%T')
+    mkdir -p ${BACKUP_DIR}
+    cp -r ${ROOT}/* ${BACKUP_DIR}
+
+    # Remove the older, modified CWS content in root dir (except backup_ folders)
+    ls ${ROOT}/ | grep -v 'bpmn'  | grep -v 'configure.sh' | grep -v 'configuration.properties' | xargs rm -r
+
+    # Replace CWS dir with clean_ cws
+    cp -a ${ROOT}/.clean_/. ${ROOT}/
```

Comment:
I see a bug here.  It will overwrite the configuration.properties in the root dir from .clean_.  So I think you need to do an rsync and exclude copying the clean dir's configuration.properties.

Follow me?

Reply:
@jamesfwood  Yes. Keeping the config.properties settings unchanged after re-runs. 

This should do it:
`rsync -av --exclude='configuration.properties' ${ROOT}/.clean_/* ${ROOT}/`